test/setup.js

```diff
@@ -13,14 +12,6 @@ const { JSDOM } = require("jsdom");
 const jsdom = new JSDOM("<!doctype html><html><body></body></html>");
 const { window } = jsdom;
 
-setLogger({
```

Comment:
Flyttet dette til testQueryClient (https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#setlogger-is-removed)